examples/vault/pyproject.toml

```diff
@@ -8,7 +8,7 @@ packages = [{include = "vault_examples"}]
 
 [tool.poetry.dependencies]
 python = "^3.7.2"
-pangea-sdk = "^3.5.0"
+pangea-sdk = { path = "../../packages/pangea-sdk", develop = true }
```

Comment:
Do we prefer to use relative path then?

Reply:
It'll be more advantageous moving forward once we have CI for the examples running. That way, having this will catch incoming changes that break examples _before_ they get published.